src/ansys/meshing/prime/autogen/surfaceutilitystructs.py

```diff
@@ -3275,3 +3275,210 @@ def error_code(self) -> ErrorCode:
     @error_code.setter
     def error_code(self, value: ErrorCode):
         self._error_code = value
+
+class TriangulateParams(CoreObject):
+    """Parameters used for triangulation operation.
```

Comment:
```suggestion
    """Provides the parameters used for triangulation operation.
```
Class descriptions should start with a verb ending in "s" or "es".